HW03/Lizaveta_Muromskaya/lizaveta_muromskaya_hw_03_t_03.rb

```diff
@@ -0,0 +1,23 @@
+def task_3(log)
+  @arr = []
+  @result = []
+  # Calling core with action:
+  a = log.split("\n")
+  a.each_with_index do |string, _index|
+    if string['Calling core with action:']
+      time = string.scan(/[0-9]+:[0-9]+:[0-9]+\.[0-9]/) # time
+    end
+    next if time.nil?
+
+    time.each do |i|
+      d = i.split(':')
+      sec = d[0].to_f * 3600 + d[1].to_f * 60 + d[2].to_f
+      @arr << sec
+    end
```

Comment:
```suggestion
    time.each do |i|
      d = i.split(':')
      sec = d[0].to_f * 3600 + d[1].to_f * 60 + d[2].to_f
      @arr << sec
    end
```
Could you move this code into another method and call it please?